public/video-ui/src/constants/videoEditValidation.js

```diff
@@ -1,5 +1,5 @@
 const validate = (values) => {
```

Comment:
I believe in this file there shoudl also be a character limit on `title`. @akash1810 - What is the limit?